### PR TITLE
Add more fake buildings

### PR DESCRIPTION
**Pull request by Amelia55555**
Improves the usefulness of the vanilla Allies country by adding high tech fake structures. At some stage I'll also let spies infiltrate fake buildings which have an infiltratable counterpart building.

### CODE REVIEW

**Comment by obrakmann** [comment]
:+1:

**Comment by MicroJOo** [comment]
:+1:

File: mods/ra/maps/allies-03a/map.yaml [hunk 1]
```diff
@@ -1546,6 +1546,15 @@ Rules:
...
 			Prerequisites: ~disabled
+	ATEF:
+		Buildable:
+			Prerequisites: ~disabled
+	MSLF:
+		Buildable:
+			Prerequisites: ~disabled
+	PDOF:
+		Buildable:
+			Prerequisites: ~disabled
```
Comment (abcdefg30) [inline mods/ra/maps/allies-03a/map.yaml]: You can remove those (and the older ones above) as you can't get an allied construction yard in allies03a.

Comment (ScottNZ) [inline mods/ra/maps/allies-03a/map.yaml]: Fixed

**Comment by MustaphaTR** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/f0f7e1a7c55d0f43ae1b23aecabd6f36847c6f95%5E...f0f7e1a7c55d0f43ae1b23aecabd6f36847c6f95)

**Pull request merged**